brd-ios/breadwallet/src/Redesign/Scenes/Buy/Main/BuyInteractor.swift

```diff
@@ -34,13 +34,11 @@ class BuyInteractor: NSObject, Interactor, BuyViewActions {
         
         dataStore?.to = 0
         
-        PaymentCardsWorker().execute(requestData: PaymentCardsRequestData()) { [weak self] result in
+        fetchCards { [weak self] result in
```

Comment:
could u not just call getPaymentCards(viewAction: .init())

and not need the private func for it?

Reply:
![image](https://user-images.githubusercontent.com/99484553/185084814-940259ab-1655-432f-9555-390667c7688a.png)
![image](https://user-images.githubusercontent.com/99484553/185084912-7fb3d113-9e8b-4a40-962d-3eb1f1c75de2.png)

they initiate different flows